### PR TITLE
[DEFECT] On www site, search results block doesn't behave nicely.

### PR DESCRIPTION
**Pull request by ziegl107**
https://trello.com/c/84FugV7d/350-defect-on-www-site-search-results-block-doesn-t-behave-nicely

### CODE REVIEW

File: app/assets/stylesheets/omni_nav/modules/omni-nav.scss [hunk 1]
```diff
@@ -176,7 +175,7 @@ html, body { margin:0; padding: 0;}
 
 
         &.expanded {
-
+            z-index: 99998;
```
Comment (tatwell) [inline app/assets/stylesheets/omni_nav/modules/omni-nav.scss]: Include a comment here explaining this value. As a rule, try to include comments for magic values like this. Here's a little more on magic numbers:

http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

File: app/assets/stylesheets/omni_nav/modules/omni-nav.scss [hunk 2]
```diff
@@ -976,11 +1009,11 @@ a.cu_nav_button {
 }
 
 .cu-search-open-trigger {
+    z-index: 99999;
```
Comment (tatwell) [inline app/assets/stylesheets/omni_nav/modules/omni-nav.scss]: I assume this value is connected with previous one. Add comment here noting that.

**Comment by mandythomas** [comment]
@tatwell in the omni-nav.scss I see Kayla added some @media statements to specify a max-width size rather than using the bp() functions from base\mixins.scss (or creating a new mixin in that file then referencing it in omni-nav.scss). Is that ok or do you want that fixed for consistency?

**Comment by tatwell** [comment]
@mandythomas is right. We should use SASS settings to keep things consistent. Thanks for pointing that out, Mandy.

**Comment by mandythomas** [comment]
looks good

**Pull request merged**